src/modules/ProtocolFeeCollector.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import { Owned, ERC20, SafeTransferLib } from "src/base/Cellar.sol";
+
+/**
+ * @title Protocol Fee Collector
+ * @notice Collects protocol fees on chains where the gravity bridge does not exist.
+ * @notice Future contracts can implement `sendToCosmos` and actually send the fees to Cosmos.
+ * @author crispymangoes
+ */
+contract ProtocolFeeCollector is Owned {
+    using SafeTransferLib for ERC20;
+
+    constructor(address _owner) Owned(_owner) {}
+
+    /**
+     * @notice Implements the Gravity Bridge `sendToCosmos` function, but
+     *         just holds ERC20s in contract.
+     */
+    function sendToCosmos(address asset, bytes32, uint256 amount) external {
```

Comment:
So in the future, we will create a real FeeCollector contract with more intricate logic. Maybe it takes fees and just sends them to cosmos through Axelar, or it could swap all the fees into WETH, and bridge it to Ethereum, then send the fees to cosmos through the gravity bridge. This is still up in the air which is why we have this temporary solution.

TLDR in the future we will have a contract that will replace this simple FeeCollector and it will actually handle the fees when `sendToCosmos` is called.